Gemfile.lock

```diff
@@ -250,25 +245,25 @@ GEM
     method_source (0.9.2)
     mime-types (3.3)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.0904)
+    mime-types-data (3.2019.1009)
     mimemagic (0.3.3)
     mini_magick (4.9.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.12.2)
+    minitest (5.13.0)
     msgpack (1.3.1)
-    mysql2 (0.5.2)
+    mysql2 (0.5.3)
     net-scp (2.0.0)
       net-ssh (>= 2.6.5, < 6.0.0)
     net-ssh (5.2.0)
     netrc (0.11.0)
     nio4r (2.5.2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
-    oj (3.9.1)
+    oj (3.10.0)
     orm_adapter (0.5.0)
-    parallel (1.17.0)
-    parser (2.6.4.1)
+    parallel (1.19.1)
```

Comment:
### [parallel](https://github.com/grosser/parallel)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.17.0 → 1.19.1

[_~~change-log~~, [source-code](https://github.com/grosser/parallel/tree/v1.19.1)_]





<details>
<summary>Commits</summary>

A change of **26** commits. See the full changes on [the compare page](https://github.com/grosser/parallel/compare/v1.17.0...v1.19.1).

These are the first 10 commits:
- (4469b03) [Remove deprecated Travis CI option](https://github.com/grosser/parallel/commit/4469b03103fbf4111fdc4a5e649e26c484a3b027)
- (c0299af) [Merge pull request #256 from RDIL/patch-1](https://github.com/grosser/parallel/commit/c0299af9b685c1a28b96e40645f3cf7b77473359)
- (c5b1c2a) [Adding additional job with RUBYOPT="--jit" for anything 2.6 …](https://github.com/grosser/parallel/commit/c5b1c2a55fabf28527e679ff4091f9d94359e234)
- (052d79b) [Added ruby_jit branch to allowed travis branches.](https://github.com/grosser/parallel/commit/052d79b39afbab942efa7e59b2299e7e113af668)
- (8efd1ed) [The --jit build had a problem with bundler caching.  I'm goi…](https://github.com/grosser/parallel/commit/8efd1ed3ab805d7c41fbf0e210c4b7f2d412c6bc)
- (cea028f) [Allow --jit jobs to fail without marking the whole build as …](https://github.com/grosser/parallel/commit/cea028fd6a55bf013b926d94fd93934e4fe10607)
- (6457ece) [Set travis back to only testing master after getting everyth…](https://github.com/grosser/parallel/commit/6457ece485345faecb9ad46b8774fc3c5ec14147)
- (acbd117) [Ubuntu precise is deprecated on Travis.  Updated to xenial.](https://github.com/grosser/parallel/commit/acbd11780195ff2bfe866b7f1cce0433c0b65290)
- (969a59b) [Switched back to precise and trying to include 2.6 JIT to th…](https://github.com/grosser/parallel/commit/969a59b112e80a89f61d7abe95172921b36c5d88)
- (da9cd60) [Merge pull request #258 from duffyjp/ruby_jit](https://github.com/grosser/parallel/commit/da9cd608d988a114e4c4f271a4f6e7ea7c47fd2f)


</details>